db/seeds/development/driver_commission_history.seeds.rb

```diff
@@ -0,0 +1,23 @@
+Octopus.using(:grossman) do
+  10.times do
+    DriverCommissionHistory.create(
+            :driver_id => Faker::Number.number(4),
+            :backhauls => 'BACKHAULS',
+            :freight_revenue => Faker::Number.number(4),
+            :customer_id => 0001,
+            :delivery_date => Date.new(2014, 01, 01),
+            :driver_rate => Faker::Number.number(4),
+            :freight_bill_number => Faker::Number.number(4),
+            :fuel_surcharge => Faker::Number.number(4),
+            :layover => 'layover',
+            :movement_type => 'MOVE TYPE',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.